components/ws-manager/go.mod

```diff
@@ -53,52 +55,52 @@ replace github.com/gitpod-io/gitpod/ws-daemon/api => ../ws-daemon-api/go // leew
 
 replace github.com/gitpod-io/gitpod/ws-manager/api => ../ws-manager-api/go // leeway
 
-replace k8s.io/api => k8s.io/api v0.20.1 // leeway indirect from components/common-go:lib
+replace k8s.io/api => k8s.io/api v0.20.4 // leeway indirect from components/common-go:lib
```

Comment:
we're trying to keep the Kubernetes dependencies consistent across the project. To that end there's a [`go-get-kubernetes.sh`](https://github.com/gitpod-io/gitpod/blob/b7dda84ad0c0dd195b829b7f4160f9f7acfa355c/components/common-go/go-get-kubernetes.sh) script, which in combination with `leeway link` updates Kubernetes dependencies.

I realise this is missing in the "How we develop" page and have updated [that page accordingly](https://www.notion.so/gitpod/How-we-develop-e8ce7e562478458a9223eb9b797415b2#f713c90fb99e4a9791232ceb646a15fb).

Reply:
fixed